recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,4 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - 10.12                  # [osx]
+c_stdlib:
+  - macosx_deployment_target   # [osx]
+c_stdlib_version:              # [osx]
+  - 10.13                      # [osx]
```

Comment:
I'm not sure why the macOS build are failing, but I also don't know if this macOS configurement stuff is needed anymore. @Kwaizer can you try a build with this file removed? (Unless you know for sure this is required)